code/game/atoms.dm

```diff
@@ -357,6 +357,7 @@ var/list/blood_splatter_icons = list()
 /atom/Stat()
 	. = ..()
 	sleep(1)
```

Comment:
intentional, I basically wanted it to make Stat()s slow down if load is higher, but not increase their rate otherwise, since stoplag() sleeps for world.tick_lag deciseconds, and that is almost always less then 1ds, if i just had that, then stats would speed up